src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-named-as-default, no-shadow, consistent-return */
+import React, { Component } from 'react';
+import { Grid, Card, Button } from 'semantic-ui-react';
+// import { connect } from 'react-redux';
+// import PropTypes from 'prop-types';
+import './Login.css';
+
+// NOTE all commented out code will be used in future mapping of state, dispatching of actions etc
+// import { renderSheets } from '../../actions/renderActions';
+// import injectWidgetId from '../../utils/utils';
+/* eslint-disable react/prefer-stateless-function */
+export class Login extends Component {
+  // componentDidMount() {
```

Comment:
these commented out lines seem to be for future development. Why not include them here? for the sake of breaking up the functionality into a number of smaller PRs? @Gwillison415